README.md

```diff
@@ -1,2 +1,17 @@
-# hw01
-Repository for homework 01
+# Homework assigment 1
+#Bio of Mary Denise Robertson
+I was born in South Africa, and came to Chicago 9 years ago to study a PhD in Anthropology. I like Chicago except the winters are *very* cold. And **very** long. 
+
+## Time at UChicago
+I like UChicago. My three favorite things are 
+
+* Naps in Harper Library
```

Comment:
Great job with the list! Your issue with spacing appears to be confined to the use of the # in making headers.